src/core/xfa/builder.js

```diff
@@ -67,6 +68,7 @@ class Empty extends XFAObject {
 class Builder {
   constructor() {
     this._namespaceStack = [];
+    this._nsAgnosticStack = [];
```

Comment:
*This is possibly a stupid question on my part, however:*
Are you planning of adding anything except simply boolean values to this Array?

If not, it may not be necessary to use an Array and an integer (e.g. `this._nsAgnosticLevel` or similar) that you just increment/decrement might work just as well!?